README.md

```diff
@@ -230,10 +230,6 @@ When a custom `Spec.Image` is used with an `OpenTelemetryCollector` resource, th
 
 We strive to be compatible with the widest range of Kubernetes versions as possible, but some changes to Kubernetes itself require us to break compatibility with older Kubernetes versions, be it because of code incompatibilities, or in the name of maintainability.
 
-Our promise is that we'll follow what's common practice in the Kubernetes world and support N-2 versions, based on the release date of the OpenTelemetry Operator.
```

Comment:
I think this has to be better discussed. Could you please open an issue to discuss this?

Reply:
Thanks, I added a couple of comments there.

Reply:
The outcome of the discussion should be documented here.  Like, "every released operator will support a specific range of Kubernetes versions, to be determined at the latest during the release."

Reply:
good idea I will add there to the readme.